release_notes/ocp-4-14-release-notes.adoc

```diff
@@ -1723,6 +1722,82 @@ Targeting {product-title} 4.15, the OpenShift SDN CNI network plugin will not be
+
+* Previously, an Agent-based installation would fail if a non-canonical IPv6 address was used for the `RendezvousIP` in the `agent-config.yaml` file. Non-canonical IPv6 addresses, for example, contain leading zeros such as `2001:0db8:0000:0000:0000:0000:0000:0000`. With this update, these valid addresses can now be used for the `RendezvousIP`. (link:https://issues.redhat.com/browse/OCPBUGS-14121[*OCPBUGS-14121*])
+
+* The Manila CSI Driver Operator automatically creates an OpenShift storage class for each available Manila share type. As part of this operation, the Operator queries the Manila API. Previously, the Operator cached the cloud credentials, which resulted in authentication issues when these credentials were rotated. Now, the Operator always uses the latest credentials. (link:https://issues.redhat.com/browse/OCPBUGS-14049[*OCPBUGS-14049*])
+
+* Previously, when configuring the `install-config.yaml` file for use during an Agent-based installation, changing the `cpuPartitioning` field to a non-default value would not produce a warning to alert users that the field is ignored for Agent-based installations. With this update, changing the `cpuPartitioning` field will cause a warning to users that the configuration will not impact the install. (link:https://issues.redhat.com/browse/OCPBUGS-13662[*OCPBUGS-13662*])
+
+* Previously, installing an Azure cluster into an existing Azure Virtual Network (VNet) might have failed because the installation program created a default network security group, which allowed traffic from `0.0.0.0`. The failure occurred when the existing VNet had the following rule enabled in the tenant: `Rule: Network Security Groups shall not allow rule with 0.0.0.0/Any Source/Destination IP Addresses - Custom Deny`. With this fix, the installation program no longer creates the default network security group when installing a cluster into an existing VNet, and the installation succeeds. (link:https://issues.redhat.com/browse/OCPBUGS-11796[*OCPBUGS-11796*])
+
+* During an installation, when the cluster status is `installing-pending-user-action` the installation won't complete until the status is resolved. Previously, if the user ran the `openshift-install agent wait-for bootstrap-complete` command there would be no indication of how to resolve the problem causing this status. With this update, the command output provides a message indicating which actions must be taken by the user to resolve the issue.
++
+As an example, the `wait-for` output when an invalid boot disk is used is now:
+[source,terminal]
```

Comment:
I think a `+` is needed above this to make it indented at the same level as the text above it.